oauth-common/src/main/java/io/strimzi/kafka/oauth/common/ConfigProperties.java

```diff
@@ -30,6 +30,13 @@ public static void resolveAndExportToSystemProperties(Properties defaults) {
         }
     }
 
+    public static void clearSystemProperties(Properties defaults) {
+        Properties p = new ConfigProperties(defaults).resolveTo(new Properties());
+        for (Object key: p.keySet()) {
+            System.clearProperty(key.toString());
+        }
+    }
+
```

Comment:
I would prefer the above method to not be part of oauth-common but just be included in the hydra-test as there it's the only place where it is used.